src/main/scala/com/avast/utils2/Done.scala

```diff
@@ -7,4 +7,6 @@ package com.avast.utils2
   */
 sealed trait Done
 
-case object Done extends Done
+case object Done extends Done {
+  def apply(): Done = this
```

Comment:
Why?

Reply:
yes, why?!

Reply:
Because it sucks to type `Done: Done`

Reply:
But doesn't this mean you have to type `Done()`? But I guess it can be added.